pkg/ccl/changefeedccl/encoder_csv.go

```diff
@@ -12,33 +12,20 @@ import (
 	"bytes"
 	"context"
 
+	"github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/cdcevent"
 	"github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeedbase"
-	"github.com/cockroachdb/cockroach/pkg/sql/catalog"
-	"github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb"
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
```

Comment:
Interesting bit after our conversation about the abstraction layer: what would it take to have zero `/sql` dependency in the encoders?

Reply:
`tree` includes datum definitions. All of the row data is given to the encoders in terms of datums. Breaking that one and `sql/types` isn't worth it.